Feature/integrate fluent form

lzzy12 opened the pull request.

hbhalodia commented:
Hi @lzzy12, Can you please rebase your PR with develop. We have added a change in PR https://github.com/rtCamp/godam/pull/564, Such that we have removed extra `if-else` statement which were being used. You can refer that PR.

lzzy12 commented:
> Hi @lzzy12, Can you please rebase your PR with develop. We have added a change in PR #564, Such that we have removed extra `if-else` statement which were being used. You can refer that PR.

Yes I am rebasing by PR

lzzy12 commented:
Moving this PR to: https://github.com/rtCamp/godam/pull/568
